external/windows64/README.md

```diff
@@ -20,7 +20,7 @@ The following applications are required when rebuilding the core C++ library, an
 For re-building the nupic.core C++ library the following are required;
 
 - [Microsoft Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs) - Community Free edition (or Enterprise edition)
-- [NumPy C++ headers](http://www.lfd.uci.edu/~gohlke/pythonlibs/) - pip install `numpy-1.9.3+vanilla-cp27-none-win_amd64.whl`
+- [NumPy C++ headers](http://www.lfd.uci.edu/~gohlke/pythonlibs/) - pip install `numpy-1.11.2+vanilla-cp27-none-win_amd64.whl`
```

Comment:
@rhyolight The MKL version from gohlke is the better version to use iff you are installing onto a device with an Intel CPU. Otherwise best to just state `pip install numpy==1.11.2` to pick up a non-optimized generic version from PyPi (and change the link to `https://pypi.python.org/pypi/numpy`).